pkg/api/types.go

```diff
@@ -1396,7 +1408,9 @@ type NodeStatus struct {
 	Conditions []NodeCondition `json:"conditions,omitempty"`
 	// Queried from cloud provider, if available.
 	Addresses []NodeAddress `json:"addresses,omitempty"`
-	// NodeSystemInfo is a set of ids/uuids to uniquely identify the node
+	// Endpoints of daemons running on the Node.
+	DaemonEndpoints NodeDaemonEndpoints `json:"daemonEndpoints,omitempty"`
+	// NodeSystemInfo is a set of ids/uuids to uniquely identify the node.
```

Comment:
I realize you didn't add this comment, but please delete "NodeSystemInfo is a". NodeSystemInfo is the Go struct type, not the field name.